README.md

```diff
@@ -9,259 +16,40 @@
-    type = string
-    default = "production"
-}
-
-variable openshift_version {
-    type = string
-    default = "4.12"
-}
-
-variable account_role_prefix {
-    type = string
-}
-
-variable token {
-    type = string
-}
-
-variable url {
-    type = string
-}
-
-terraform {
-  required_providers {
-    aws = {
-      source  = "hashicorp/aws"
-      version = ">= 4.20.0"
-    }
-    ocm = {
-      version = "0.0.3"
-      source  = "terraform-redhat/ocm"
-    }
-  }
-}
-
-provider "ocm" {
-  token = var.token
-  url = var.url
-}
-
-data "ocm_policies" "all_policies"{}
-
-module "create_account_roles"{
-  source = "terraform-redhat/rosa-sts/aws"
-  version = "0.0.4"
-
-  create_operator_roles = false
-  create_oidc_provider = false
-  create_account_roles = true
-
-  account_role_prefix =  var.account_role_prefix
-  ocm_environment =  var.ocm_environment
-  rosa_openshift_version=  var.openshift_version
-  account_role_policies=data.ocm_policies.all_policies.account_role_policies
-  operator_role_policies=data.ocm_policies.all_policies.operator_role_policies
-}
-
-```
-
-* Least AWS Permissions required to run the terraform
-
-```
-{
-    "Version": "2012-10-17",
-    "Statement": [
-        {
-            "Sid": "VisualEditor0",
-            "Effect": "Allow",
-            "Action": [
-                "iam:GetRole",
-                "iam:UpdateOpenIDConnectProviderThumbprint",
-                "iam:CreateRole",
-                "iam:DeleteRole",
-                "iam:UpdateRole",
-                "iam:DeleteOpenIDConnectProvider",
-                "iam:GetOpenIDConnectProvider",
-                "iam:CreateOpenIDConnectProvider",
-                "iam:TagOpenIDConnectProvider",
-                "iam:TagRole",
-                "iam:ListRolePolicies",
-                "iam:ListAttachedRolePolicies",
-                "iam:ListInstanceProfilesForRole",
-                "iam:AttachRolePolicy",
-                "iam:DetachRolePolicy"
-            ],
-            "Resource": [
-                "arn:aws:iam::<ACCOUNT_ID>:oidc-provider/*",
-                "arn:aws:iam::<ACCOUNT_ID>:role/*"
-            ]
-        }
-    ]
-}
-```
-
-* Choose operator IAM roles prefix name
-
-The operator IAM roles will be created per cluster by [module](https://registry.terraform.io/modules/terraform-redhat/rosa-sts).
-
-
-## Sample Terraform Manifest File
+1. [Create a ROSA cluster that usess STS and has a managed OIDC configuration](https://github.com/terraform-redhat/terraform-provider-ocm/blob/529b10c22c13810b3edbaa01327c7dfdcc207650/examples/create_rosa_cluster/create_rosa_sts_cluster/oidc_configuration/cluster_with_managed_oidc_config/README.md)
+1. [Create a ROSA cluster that uses STS and has an unmanaged OIDC configuration](https://github.com/terraform-redhat/terraform-provider-ocm/blob/529b10c22c13810b3edbaa01327c7dfdcc207650/examples/create_rosa_cluster/create_rosa_sts_cluster/oidc_configuration/cluster_with_unmanaged_oidc_config/README.md)
```

Comment:
Now you don't have the link to the create account, so maybe can add it here:
1. pre cluster creation examples
(account roles)
2. Cluster creation examples
(managed and unmanaged clusters you already added)
3. post cluster creation examples
(We can have here the Adding identity providers examples)